src/suite-config.rst

```diff
@@ -2281,6 +2281,9 @@ The task job script may export the following environment variables:
    CYLC_VERSION                    # Version of cylc installation used
 
    CYLC_CYCLING_MODE               # Cycling mode, e.g. gregorian
+   ISODATETIMECALENDAR             # Calendar for the `isodatetime`,
+                                   # defined with the value of CYLC_CYCLING_MODE
+                                   # in a date-time cycling mode
```

Comment:
Perhaps note that this is for the `isodatetime` command.

Reply:
Done.